### PR TITLE
Expand Ansible Tower to support OCP 4.18

### PR DESCRIPTION
**Pull request by madunn**
Expanded Ansible Tower to support 4.18 and 4.18 fips

### CODE REVIEW

File: ci-operator/config/ansible/tower-qa/ansible-tower-qa-devel__ocp4.18-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aap-interop-aws
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/config/ansible/tower-qa/ansible-tower-qa-devel__ocp4.18-lp-interop.yaml:25]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/config/ansible/tower-qa/ansible-tower-qa-devel__ocp4.18-lp-interop.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        ]
+      USER_TAGS: |
+        scenario aap
+    test:
+    - ref: install-operators
+    - ref: aap-install-automation-controller
+    - ref: aap-tests-integration
+    workflow: firewatch-ipi-aws
+- as: aap-interop-aws-fips
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/config/ansible/tower-qa/ansible-tower-qa-devel__ocp4.18-lp-interop.yaml:54]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/jobs/ansible/tower-qa/ansible-tower-qa-devel-periodics.yaml [hunk 3]
```diff
@@ -553,3 +553,188 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build05
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/jobs/ansible/tower-qa/ansible-tower-qa-devel-periodics.yaml:558]:
```suggestion
  cron: 0 0 11 6 *
```

File: ci-operator/jobs/ansible/tower-qa/ansible-tower-qa-devel-periodics.yaml [hunk 4]
```diff
@@ -553,3 +553,188 @@ periodics:
...
+        secretName: manifest-tool-local-pusher
+    - name: pull-secret
+      secret:
+        secretName: registry-pull-credentials
+    - name: result-aggregator
+      secret:
+        secretName: result-aggregator
+- agent: kubernetes
+  cluster: build05
+  cron: 0 6 11 2 *
```
Comment (dfrazzette) [inline ci-operator/jobs/ansible/tower-qa/ansible-tower-qa-devel-periodics.yaml:650]:
```suggestion
  cron: 0 0 11 6 *
```

**Comment by madunn** [comment]
Closing as duplicate of https://github.com/openshift/release/pull/57603

**Pull request closed without merging**